packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -0,0 +1,601 @@
+<!-- eslint-disable vue/no-restricted-class -->
+<template>
+  <div
+    data-qa="dt-editor"
+    role="presentation"
+    :class="['d-d-flex', 'd-fd-column', 'd-baw1', 'd-ba', {
+      'd-bc-black-500': hasFocus,
+      'd-bc-default': !hasFocus,
+      'd-bar8': roundedEdges,
+      'd-bar0': !roundedEdges,
+    }]"
+    @click="$refs.richTextEditor.focusEditor()"
+  >
+    <!-- Section for the top UI -->
+    <dt-stack
+      direction="row"
+      gap="100"
+      :class="['d-px8', 'd-py2', 'd-bgc-black-200', {
+        'd-btr8': roundedEdges,
+        'd-btr0': !roundedEdges,
+      }]"
+    >
+      <dt-button
+        v-if="showBoldButton"
+        data-qa="dt-editor-bold-btn"
+        :class="['h:d-bgc-black-300', 'd-fc-black-700', 'd-mx4', {
```

Comment:
We can remove the margins now that we have dt-stack with gap (increase gap if necessary)